pyproject.toml

```diff
@@ -25,7 +25,7 @@ classifiers = [
     'Programming Language :: Python :: Implementation :: PyPy',
     'Topic :: Software Development',
 ]
-include = ["README.rst", "CHANGELOG.md", "pylint_venv.py"]
+include = ["pylint_venv.py"]
```

Comment:
Now `CHANGELOG.md` is not included in sdist (README is included by Poetry implicitly). If you wanted to preserve the earlier behavior, you could have done:

```
include = [
    ...
    { path = "CHANGELOG.md", format = "sdist" },
]
```